list.go

```diff
@@ -0,0 +1,83 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tPID\tSTATUS\tSTARTED\n")
```

Comment:
Cause I originally called the variable container_timestamp with an output at the runc list command column name of started so as to indicate the time when the container was started.  Then vish wanted to call the variable creation_timestamp... then dug wanted to call the variable creation_time.  But you're the first one to ask about the output column heading name.  :-)  

All that said I have no druthers what column heading would you like to see?   "CREATION TIME"  does not really roll off the tongue.